flask/todo_app/services/todo_service.py

```diff
@@ -0,0 +1,13 @@
+from todo_app.lib.model_manager import SqlAlchemyModelManager
+from todo_app.models.todo_models import Todo
+from todo_app.services import Service
+
+__all__ = ['TodoService']
+
+
+class TodoManager(SqlAlchemyModelManager):
```

Comment:
I think managers should be separate.  It is possible that multiple services will leverage the same manager.  i.e. to handle service composition or just things that require transactional integrity.